pkg/handler/handler.go

```diff
@@ -178,6 +182,14 @@ func (o *OpenAPIService) RegisterOpenAPIVersionedService(servePath string, handl
 							return
 						}
 					}
+					// Set Content-Type header in the reponse
+					if accepts.SubType != "com.github.proto-openapi.spec.v2@v1.0+protobuf" {
+						contentType := accepts.Type + "/" + accepts.SubType
+						w.Header().Set("Content-Type", contentType)
+					} else {
+						contentType := accepts.Type + "/" + "com.github.proto-openapi.spec.v2.v1.0+protobuf"
+						w.Header().Set("Content-Type", contentType)
+					}
```

Comment:
I found this confusing to read, probably because positive conditions are easier to read than negatives
```suggestion
					contentType := accepts.Type + "/" + accepts.SubType				
					if accepts.SubType == "com.github.proto-openapi.spec.v2@v1.0+protobuf" {
						// Replace invalid mime-type in response Content-Type with proper value.
						contentType := accepts.Type + "/" + "com.github.proto-openapi.spec.v2.v1.0+protobuf"
					}
					w.Header().Set("Content-Type", contentType)
```

Reply:
changed check to positive in if